src/sentry/search/events/types.py

```diff
@@ -199,6 +199,13 @@ def organization_id(self) -> int | None:
             return self.organization.id
         return None
 
+    @property
+    def rpc_organization_id(self) -> int:
```

Comment:
Lets change this name, the start & end reference `rpc` because they return a proto object

Reply:
skipped this change due to the next comment